telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,39 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 64-bit values are inserted after IP TCP/UDP 
+header to indicate the existence of INT after TCP/UDP. These fields should be 
+interpreted as unsigned integer values, stored in network byte order and are 
+initialized to a configured value.  This Probe marker is a variation of an early 
+IETF draft with existing implementations. More information can be found at 
+https://tools.ietf.org/html/draft-lapukhov-dataplane-probe-01.
 
-The decision to use DSCP or destination port field and the reserved values are
-per domain decisions that all INT capable devices should follow. (Note: in
+INT probe marker for TCP/UDP:
+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (1)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |                         Probe Marker (2)                      |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+The decision to use DSCP, destination port field or probe marker option
+are per domain decisions that all INT capable devices should follow. The
+same applies to the reserved value of DSCP or port number, and the 
+configured probe marker value, which should be consistent within a domain.
+It is recommended that only one option is deployed per domain. (Note: in
 general, encoding into DSCP field will be less intrusive compared to changing
 the layer 4 port field. The latter may alter ECMP behavior and can complicate
-ACL and network debugging.)
+ACL and network debugging. If DSCP field cannot be reserved for INT, and 
+retaining ECMP behavior is desired, probe marker option could be followed.
+With arbitrary values being inserted after TCP/UDP header in probe marker
+option, the likelihood of conflicting with user traffic in a data center is 
+low, but cannot be completely eliminated. To further reduce the chance of 
+conflict, a user deploys probe marker option could choose to check more into
```

Comment:
the user who deploys probe marker 
or
the user deploying probe marker

It is not clear what is the "port number" referring to. You may want to change it to:
check more into the TCP/UDP destination port number

Reply:
Changed it to "the user who deploys probe marker option could choose to check more
into TCP/UDP port numbers to identify the existence of INT."  I want to keep the flexibility to check either source or destination port.